docs/source/federation-versions.mdx

```diff
@@ -0,0 +1,471 @@
+---
+title: Federation version changelog
+description: Understand changes between federation versions
+---
+
+## v2.3
+
+**Additions**
+
+[`@interfaceObject`](https://www.apollographql.com/docs/federation/federated-types/interfaces) a directive that allows you to abstract an interface in isolation by declaring it locally as an object type. This enables you to contribute fields without needing to redefine all the interface’s concrete implementation types.
```

Comment:
should those be bulleted items? i.e. it is unclear whether `@key` on interface definitions is separate item or not?